components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -439,14 +465,96 @@ public void saveBusinessOwner(BusinessOwner businessOwner){
                 try {
                     connection.rollback();
                 } catch (SQLException e1) {
-                    log.error("Failed to rollback the add entitlement policy partial with name : ", e1);
+                    handleException("Failed the rollback of save business owner with name : " + businessOwnerName, e1);
                 }
             }
         } finally {
             APIMgtDBUtil.closeAllConnections(statementToInserBusinessOwner, connection, null);
         }
+        return businessOwnerId;
     }
 
+    /**
+     * Search business owners.
+     * @param startIndex
+     * @param pageSize
+     * @param searchValue
+     * @return
+     * @throws AppManagementException
+     */
+    public List<BusinessOwner> searchBusinessOwners(int startIndex, int pageSize, String searchValue) throws
+                                                                                              AppManagementException {
+        Connection connection = null;
+        PreparedStatement statementToGetBusinessOwners = null;
+        List<BusinessOwner> businessOwnersList = new ArrayList<BusinessOwner>();
+        ResultSet businessOwnerResultSet = null;
+
+        try {
+            connection = APIMgtDBUtil.getConnection();
+            String queryToGetBusinessOwner = "SELECT * FROM BUSINESS_OWNER WHERE OWNER_NAME LIKE ? OR " +
+                    "OWNER_EMAIL LIKE ? OR OWNER_SITE LIKE ? OR OWNER_DESC LIKE ? ";
+            if (connection.getMetaData().getDriverName().contains("Oracle")) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5